events/event.go

```diff
@@ -13,9 +11,15 @@ type Event struct {
 	Height     int64
 }
 
+// ABCIEventWithHeight adds a height field to abci.Event
+type ABCIEventWithHeight struct {
+	Height int64
+	abci.Event
+}
+
 // Parse parses string to event
-func Parse(event abci.Event) (Event, error) {
-	e := Event{Type: event.Type, Attributes: make(map[string]string)}
+func Parse(event ABCIEventWithHeight) Event {
```

Comment:
```suggestion
func (event ABCIEventWithHeight) ToEvent() Event {
```

I think "Parse" is a bit of a misnomer here -- it seems more like a mapping function to me

Reply:
I used your `Map` suggestion. I need to preserve the signature, otherwise I need to refactor more in other places. Since `Event` is deprecated I don't think I should put more effort into this than necessary

Reply:
oh, right, great point